src/test/java/com/google/api/generator/gapic/composer/goldens/BatchingDescriptorComposerTestSubresponse.golden

```diff
@@ -56,4 +56,4 @@ public long countBytes(PublishRequest request) {
 return request.getSerializedSize();
 }
 
-}
+}
```

Comment:
This is intended since the generated Batchingdescriptor code snippet is without new line.